content/en/docs/03/_index.md

```diff
@@ -155,7 +155,7 @@ NAME                              READY   STATUS    RESTARTS   AGE
 example-web-go-69b658f647-xnm94   1/1     Running   0          39s
 ```
 
-The Deployment defines that one replica should be deployed --- which is running as we can see in the output. This Pod is not yet reachable from outside of the cluster.
+The Deployment defines that one replica should be deployed --- which is running as we can see in the output. This Pod is not yet reachable from outside the cluster.
```

Comment:
Don't agree. See https://dict.leo.org/german-english/outside%20of vs. https://dict.leo.org/german-english/outside

Reply:
outside of: This phrase is redundant. Consider using outside.

Reply:
I don't see where the phrase should be redundant. According to dict.leo.org, "outside of" means "ausserhalb", "outside" means "aussen".